biorxiv/corpora_comparison/biorxiv_pmc_figure_generator.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Tree Plots for bioRxiv and Pubmed Central
+
+# In[1]:
+
+
+import pandas as pd
+
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+
+
+# # bioRxiv vs Pubmed Central
+
+# In[2]:
+
+
+full_text_comparison = pd.read_csv("output/full_corpus_comparison_stats.tsv", sep="\t")
+full_text_comparison.head()
+
+
+# In[3]:
+
+
+full_text_comparison = (
+    full_text_comparison
+    .assign(repository=lambda x: x.odds_ratio.apply(lambda ratio: "bioRxiv" if ratio > 1 else "PMC"))
+    .sort_values("odds_ratio", ascending=False)
+)
+full_text_comparison
+
+
+# In[4]:
+
+
+subset = 20
+plot_df = (
```

Comment:
Can you remind me what _total counts are? Are these the total number of occurrences of all words, thats why they are the same for each word. So your odds ratio is the enrichment of the occurrences of a given word in that repo compared to all words occurring?

Reply:
>Are these the total number of occurrences of all words, that's why they are the same for each word. 

Correct. These are the total number of words found within a given corus.

>So your odds ratio is the enrichment of the occurrences of a given word in that repo compared to all words occurring?

Almost the odds ratio is measuring the association of the chance that a given word appears in one corpus rather than another. For example a ratio of >1 would indicate a word appears more often in bioRxiv rather than in PubMed Central. Likewise ratio <1 would indicate that a word appears more in PubMed Central than bioRxiv

Reply:
That makes sense i see, thanks